sections/basics/extending-styles.md

```diff
@@ -75,7 +75,7 @@ const Button = styled.button`
   display: block;
 `;
 
-const ReversedButton = props => <Button {...props} children={props.children.split('').reverse()} />
+const ReversedButton = props => <Button {...props} children={props.children.split('').reverse().join('')} />
```

Comment:
I think we can extract it into utils:

```
const reveseString = str => str.split('').reverse().join('')
```

or use a library?

Reply:
![image](https://user-images.githubusercontent.com/18689106/97797581-8885d180-1c1e-11eb-904f-2f59158614b4.png)

That's right. Does this work?

Reply:
It turned out to be just a doc, so just pretend I haven't said. lol